### PR TITLE
Added Timestamp when a new message is sent 

### LINKED ISSUE #35: Add Timestamp when a new message is sent
**Is your feature request related to a problem? Please describe.**
I want to add a new feature. When a user send a message the time of that message is send shown in chat box like whatsapp or other messaging apps.
**Describe the solution you'd like**
I want to add timestamp of message.

**Additional context**
Please assign it under LGMSOC21.

### PR DESCRIPTION
**Pull request by Veeresh-Rex**
Fixes #35 

#### Short description of what this resolves:
It adds time when a new message is sent.

#### Checklist

- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added tests that prove my fix is effective or that my feature works
Screenshot 2021-06-17 091208

### CODE REVIEW

File: socketServer.js [hunk 1]
```diff
@@ -10,13 +10,21 @@ const sock = socket => {
...
     });
-
+    const date = new Date();
+    let minute = date.getMinutes();
+    if (minute <= 9) {
+      minute = '0' + minute;
+    }
+    const newTime = `${date.getHours()}:${minute}`;
+    // eslint-disable-next-line no-console
+    console.log(newTime);
```
Comment (Creatoon) [inline socketServer.js]: Remove console.log

Comment (Veeresh-Rex) [reply socketServer.js]: done

**Comment by Creatoon** [comment]
@Veeresh-Rex  you are changing the indentation of code, please disable any code formatter , and update this PR

**Comment by Veeresh-Rex** [comment]
> 
> 
> @Veeresh-Rex you are changing the indentation of code, please disable any code formatter , and update this PR
Done, please check

**Comment by Creatoon** [comment]
@Veeresh-Rex You have done an amazing work, be consistent.

**Pull request merged**